sormas-backend/src/main/java/de/symeda/sormas/backend/immunization/ImmunizationFacadeEjb.java

```diff
@@ -455,7 +463,30 @@ public Boolean linkRecoveryImmunizationToSearchedCase(String specificCaseSearchV
 			return true;
 		}
 		return false;
-	};
+	}
+
+	protected void updateVaccinationStatuses(Immunization updatedImmunization, ImmunizationDto currentImmunization) {
```

Comment:
We should somehow inform users about this mechanism. Especially since it only set's the status to "vaccinated" and not vice-versa.